ruff.toml

```diff
@@ -0,0 +1,76 @@
+# Enable rules.
+select = [
+    "A",
+    "C4",
+    "C90",
+    "COM",
+    #"D", # pydocstyle - not enabled temporarily
+    "E",
+    "ERA",
+    "FA",
+    "FIX",
+    "F",
+    "I",
+    "ICN",
+    "ISC",
+    "PLE",
+    "PLC",
+    "PLR",
+    "PLW",
+    "PL",
+    "PYI",
+    "Q",
+    "PTH",
+    "TD",
+    "W"
+]
+
+# Ignore specific rules
+ignore = [
+    "W291", # https://beta.ruff.rs/docs/rules/trailing-whitespace/
+    "PLR0913", # https://beta.ruff.rs/docs/rules/too-many-arguments/
+    "PLR2004", #https://beta.ruff.rs/docs/rules/magic-value-comparison/
+    "PLW0603", #https://beta.ruff.rs/docs/rules/global-statement/
+    "COM812", # Trailing comma missing - disabled temporarily
+    "PLC1901", # Compare-to-empty-string - disabled temporarily
+    "ERA001", # Found commented-out code - disabled temporarily
+    "PLW", # Warning category - disabled temporarily
+    "PLR", # Refactoring category - disabled temporarily
+    "PLC", # Convention category - disabled temporarily
+    "PLE", # Error category - disabled temporarily
+    "ISC", # flake8-implicit-str-concat - disabled temporarily
+    "E501", # Line too long - disabled temporarily
+    "I001", # isort - disabled temporarily
+    ]
+
+# Exclude files and directories
+exclude = ["docs", ".eggs", "setup.py", "example", ".aws-sam", ".git", "dist", ".md", ".yaml", "example/samconfig.toml", ".txt", ".ini"]
+
+# Maximum line length
+line-length = 120
+
+
+#fix = true
+#fixable = ["I", "COM812", "W"]
```

Comment:
After fixing all previous errors, we can enable automatic fix for specific checks.